scripts/alcf_mira_cmake

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+EXTRA_ARGS=$@
+BLAS_DIR=/soft/libraries/alcf/20130312/gcc/BLAS
+LAPACK_DIR=/soft/libraries/alcf/20130312/gcc/LAPACK
+BOOST_DIR=/soft/libraries/boost/1.58.0/cnk-gcc/V1R2M2-20150523
+TRILINOS_DIR=/home/lebrung/projects/dtk/source/trilinos/12.8.1
+ARGS=(
+    -D CMAKE_BUILD_TYPE=Debug
+    -D BUILD_SHARED_LIBS=OFF
+    -D TPL_FIND_SHARED_LIBS=OFF
+    ### SETUP TO RUN MPI TESTS ###
+    -D MPI_EXEC=runjob
+    -D MPI_EXEC_NUMPROCS_FLAG="--np"
+    -D MPI_EXEC_PRE_NUMPROCS_FLAGS=""
+    -D MPI_EXEC_POST_NUMPROCS_FLAGS="--block \$BLOCKID :"
```

Comment:
For future references where did you find the answer? Please point to the documentation.